sepal_ui/aoi/aoi_model.py

```diff
@@ -621,7 +621,7 @@ def get_ipygeojson(self, style: Optional[dict] = None) -> GeoJSON:
         # adapt the style to the theme
         if style is None:
             style = json.loads((ss.JSON_DIR / "aoi.json").read_text())
-            style.update(color=color.success, fillColor=color.success)
+            style.update(color=color.primary, fillColor=color.primary)
```

Comment:
but isn't it redundant? `add_layer` adds a default style when `GeoJSON` doesn't have one